lib/mvi/configuration.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+require 'common/client/configuration/soap'
+
+module MVI
+  class Configuration < Common::Client::Configuration::SOAP
+    URL = ENV['MVI_URL']
+    OPEN_TIMEOUT = ENV['MVI_OPEN_TIMEOUT']&.to_i || 2
```

Comment:
Rather than silently defaulting, logging a warning could be beneficial.

Example:
```ruby
OPEN_TIMEOUT = ENV['MVI_OPEN_TIMEOUT']&.to_i || default_mvi_open_timeout
def default_mvi_open_timeout
  logger.warn 'MVI_OPEN_TIMEOUT env variable not set, using default'
  2
end
```